test/core/consensus/grandpa/literals.hpp

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_TEST_CORE_CONSENSUS_GRANDPA_LITERALS_HPP
+#define KAGOME_TEST_CORE_CONSENSUS_GRANDPA_LITERALS_HPP
+
+#include "consensus/grandpa/vote_weight.hpp"
+
+inline kagome::consensus::grandpa::VoteWeight makeVoteWeight(uint64_t s) {
+  // TODO(warchant): update this when VoteWeight is updated
```

Comment:
Maybe reassign to somebody else, as warchant actively participated in the project no more.

Reply:
Will be discussed offline

Reply:
name here does not mean who should do it, but who reported that